CONTRIBUTING.md

```diff
@@ -127,16 +127,6 @@ Anything that does not cause user or downstream dependency regressions can be ba
 - Services which require a client to be up-to-date regardless. (E.g. `spotify`, `steam`, or `discord`)
 - Security critical applications (E.g. `firefox`)
 
-## Generating 23.05 Release Notes
```

Comment:
The pull request template links to this anchor. We should at least keep the path `nixos/doc/manual/release-notes/rl-2305.section.md` somewhere.